### DISCUSSION TITLE
Unable to draw text on a canvas with Hololens 2

### DISCUSSION THREAD

**Discussion by artificial-jon** [servo/servo#27494] [General] [answered]
Nothing draws when I try to draw text into a canvas using canvas.fillText().  Here is a code snipped that works on chrome but doesn't draw anything for me with FxR on Hololens 2.  I assume this should work.  Anything I can try to provide more information?

[codepen](https://codepen.io/jfournie/pen/bGpGGgY)

```
const testCanvas = document.createElement('canvas');
testCanvas.width = 256;
testCanvas.height = 256;
const ctx = testCanvas.getContext('2d');
ctx.fillStyle = '#FFF';
ctx.fillRect(0, 0, testCanvas.width, testCanvas.height);
ctx.font = "30px Comic Sans MS";
ctx.fillStyle = "red";
ctx.textAlign = "center";
ctx.fillText("Hello World", testCanvas.width / 2, testCanvas.height / 2);

document.body.appendChild(testCanvas);
testCanvas.style.position = 'absolute';
testCanvas.style.top = '200px';
testCanvas.style.left = '500px';
```

**Comment by jdm** [comment, marked as answer]
The changes to support drawing text on canvas under UWP merged as part of https://github.com/servo/servo/pull/27207. Unfortunately that same PR also introduced a startup regression that is being tracked in https://github.com/servo/servo/issues/27492.
